tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -100,6 +131,8 @@ public function addProductToOrderWithFreeShippingAndNewInvoice(
 
     /**
      * @When I generate invoice for :invoiceReference order
+     *
+     * @param $orderReference
```

Comment:
annotation is useless if you don't specify anything more than method signature does